pkg/api/server/docs.go

```diff
@@ -4,6 +4,24 @@
 // only as experimental as this point.  The endpoints, parameters, inputs, and
 // return values can all change.
 //
+// To start the service and keep it running for 5,000 seconds (-t 0 runs forever):
+//    podman system service -t 5000 &
+//
+// You can then use cURL on the socket using requests documented below.
+//
+// NOTE: if you install the package podman-docker, it will create a symbolic
+// link for /var/run/docker.sock to /run/podman/podman.sock
+//
+// See podman-service(1) for more information.
+//
+//  Quick Examples:
+//   'podman info'
+//      curl --unix-socket /run/podman/podman.sock http://d/v1.0.0/libpod/info
```

Comment:
For a TCP connection the example might be:
```
curl --silent localhost:8080/v1.0.0/libpod/info 
```

Reply:
I would really recommend that people not use tcp, since there is NO Security wrapper around this.  It is basically opening a welcome to root user without a password if you listen on a network port and run as root.  Running as non root, gives away access to your homedir.

Reply:
Thanks for the security hint, you are totally right.

Reply:
If I start podman system service as user from the example above (w/ or w/o timeout)
```
$ podman system service 
```